cellprofiler/modules/identifysecondaryobjects.py

```diff
@@ -149,6 +158,7 @@
 
 """.format(**{
     "DEFINITION_OBJECT": _help.DEFINITION_OBJECT,
+    "HELP_ON_SAVING_OBJECTS": _help.HELP_ON_SAVING_OBJECTS
```

Comment:
I don't see any corresponding `{HELP_ON_SAVING_OBJECTS}` in the documentation string. This line should probably be added above "Measurements made by this module", like in IdentifyPrimaryObjects.